automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -xe
+
+# This script should be able to execute KubeMacPool
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-kubemacpool-functests.sh
+
+teardown() {
+    make cluster-down
+    rm -rf "${TMP_COMPONENT_PATH}"
+}
+
+main() {
+    export COMPONENT="kubemacpool"
+
+    # Spin-up ephemeral cluster with all necessary components
+    source automation/e2e-functests.setup.sh
+
+    cd ${TMP_COMPONENT_PATH}
+
+    # Deploy necessary network addons components with CNAO
+    cat <<EOF > cr.yaml
+apiVersion: networkaddonsoperator.network.kubevirt.io/v1alpha1
+kind: NetworkAddonsConfig
+metadata:
+  name: cluster
+spec:
+  imagePullPolicy: Always
+  kubeMacPool: {}
+  linuxBridge: {}
+  multus: {}
+EOF
+    kubectl apply -f cr.yaml
+    kubectl wait networkaddonsconfig cluster --for condition=Available --timeout=13m
+
+    # Run KubeMacPool functional tests
+     KUBECONFIG=${KUBECONFIG} E2E_TEST_ARGS="--ginkgo.skip='mac-pool is full'" make functest
```

Comment:
nit: ident

Reply:
What do you mean?

Reply:
line 41 has one space more than it needs, isnt it ?

Reply:
Gotcha, I will fix it in the following commits